examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -236,8 +236,16 @@ extern "C" void app_main()
     chip::rpc::Init();
 #endif
 
+    uint8_t features = static_cast<uint8_t>(chip::app::Clusters::DeviceEnergyManagement::sFeatureMap.Raw());
```

Comment:
Should have a comment explaining why this cast to uint8_t is safe.  But better yet would be to just not do that casting and use the actual type involved here, no?

Reply:
I've reworked to avoid any casting and printed it as a full int32_t instead.